src/sagemaker/workflow/pipeline.py

```diff
@@ -542,6 +542,35 @@ def update_args(args: Dict[str, Any], **kwargs):
 class ImmutablePipeline(Pipeline):
     """ImmutablePipeline to support pipelines that should be immutable."""
 
+    def display(self, pipeline_arn: str = None):
+        """Prints out a Graphviz DAG visual for the Pipeline
+
+        Args:
+        pipeline_arn (str): The pipeline arn for the desired pipeline.
```

Comment:
These doc strings will be auto rendered to a public facing website. So we should make it more informative.
How about:
```
Args:
            pipeline_arn (str): The pipeline ARN for the desired pipeline to display
```
Please correct it if I made any grammar mistakes

Reply:
Don't forget the indent here, ahead of `pipeline_arn`. Otherwise the docstring may not be correctly rendered.